extensions/frontend-tiles/src/integration-test/utils/webpack-config.js

```diff
@@ -0,0 +1,131 @@
+// /*---------------------------------------------------------------------------------------------
+//  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+//  * See LICENSE.md in the project root for license terms and full copyright
+//  *notice.
+//  *--------------------------------------------------------------------------------------------*/
+
+const path = require('path');
+const {globSync} = require('glob');
+const frontendLib = path.resolve(__dirname, '../../../lib/cjs');
+const Dotenv = require('dotenv-webpack');
+const webpack = require('webpack');
+const copyExternalsPlugin =
```

Comment:
you need to add a dev dep on this and import it from the pkg not this super deep import

Reply:
I believe that should be fixed now, but same behavior as my previous comment.